qtodotxt/lib/task_htmlizer.py

```diff
@@ -38,16 +39,27 @@ def task2html(self, task):
             html += ' <font color="gray">(created: {})</font>'.format(task.creation_date)
         return html
 
+    def _addUrl(self, word, color="none"):
+        if "://" in word:
```

Comment:
Can you move the test outside of the method? I think it is clearer. Otherwise all good! Great job